typings/index.d.ts

```diff
@@ -630,7 +630,7 @@ export class GuildAuditLogs {
   public toJSON(): unknown;
 }
 
-export class GuildAuditLogsEntry {
+declare class GuildAuditLogsEntry {
```

Comment:
While I agree that the class isn't *exported* by us directly (maybe it should be..), it should still be available to TS users. Thoughts @discordjs/the-big-3?

Reply:
I think we should just export it.

Reply:
@SpaceEEC Just in the typings or actually exported?

Reply:
I'd say actually exported.